src/test/resources/bulk/fms/05_expression/temp.json

```diff
@@ -0,0 +1,33 @@
+[
```

Comment:
Presumably this file shouldn't have been committed?

Reply:
Removed!